openeo/util.py

```diff
@@ -375,6 +375,21 @@ def deep_set(data: dict, *keys, value):
         raise ValueError("No keys given")
 
 
+def guess_format(outputfile: str = None):
+    """
+    Guess the output format from a given filename and return the corrected format.
+    Any names not in the dict get passed through in upper case.
+    """
+    format_string = outputfile.rsplit(".", 1)[-1].upper()
+
+    format_map = {
+        "GTIFF": "GTIFF", "GEOTIFF": "GTIFF", "GEOTIF": "GTIFF", "TIFF": "GTIFF", "TIF": "GTIFF",
+        "NC": "NetCDF", "NETCDF": "NetCDF",
+    }
```

Comment:
some more nitpicking :smile:  :

- according to https://gdal.org/drivers/raster/index.html  it's `netCDF`, not `NetCDF`
- if you use lowercase for `format_string` (which I would rename to `extension` for clarity), the dict will be more readable: keys in lowercase, values in upper case (mostly)

Reply:
and apparently it's also `GTiff` instead of `GTIFF`